examples/search/analysis.ini

```diff
@@ -179,10 +179,10 @@ statistic-files = ${resolve:./statHL.hdf} ${resolve:./statLV.hdf} ${resolve:./st
 timeslide-interval = 0.1
 
 [coinc-full_data-2det]
-loudest-keep-values = 0:25
+loudest-keep-values = [0:25]
```

Comment:
Maybe try with a negative value also now to demonstrate this works? (and have it be tested now)

Reply:
I've put it in with negative values to check it

Reply:
That check has passed now